### PR TITLE
Articles Starting with U: Rename filenames, add titles

### PR DESCRIPTION
**Pull request by Rafase282**
Refer #889

### CODE REVIEW

File: SASS.md [hunk 1]
```diff
@@ -91,61 +89,63 @@ article {
...
 header {
   background-color: $main-color;
 }
 ```
 
-In this code snippet, we have assigned the value of the header's ```background-color``` to the value stored in ```$main-color```, which (when Sass compiles to CSS) is output as:
+In this code snippet, we have assigned the value of the header's `background-color` to the value stored in `$main-color`, which (when Sass compiles to CSS) is output as:
 
-```
+```sass
```
Comment (abhisekp) [inline SASS.md]: ~~sass~~ -> scss

File: Ubuntu-Customising.md [hunk 2]
```diff
@@ -4,30 +4,32 @@ This tutorial shows you how to add aliases to the terminal, tweaking unity and r
...
 To install the Unity Tweak Tool type `sudo apt install unity-tweak-tool`, and to launch it, `unity-tweak-tool`.
 
 [Here](http://www.techrepublic.com/blog/linux-and-open-source/six-must-have-ubuntu-unity-tweaks/) is a list of the six must-have Ubuntu Unity Tweaks.
 
-:point_left: Previous | :book: Home :book: |  Next :point_right:
-:---|:---:|---:
- [**Download and Install Ubuntu Desktop**](Ubuntu-Download-and-Install) | [**Table of Contents**](Setting-Up-Ubuntu-For-Programming) | [**Jazzing up the Terminal**](Ubuntu-Jazzing-up-the-Terminal)
\ No newline at end of file
+:point_left: Previous                                                  |                     :book: Home :book:                     |                                            Next :point_right:
+:--------------------------------------------------------------------- | :--------------------------------------------------------: | ------------------------------------------------------------:
```
Comment (abhisekp) [inline Ubuntu-Customising.md]: ughh... :stuck_out_tongue_winking_eye:

Comment (Rafase282) [inline Ubuntu-Customising.md]: Seems to render fine both here and in atom.

Comment (abhisekp) [inline Ubuntu-Customising.md]: @Rafase282 but it was fine before. Looks messy now. :pensive:

File: Ubuntu-Downloading-Text-Editors-and-IDEs.md [hunk 3]
```diff
@@ -62,13 +64,13 @@ You can now drag this file to the launcher, enabling you to start Eclipse.
...
 sudo mv idea-2016.1.1.tar.gz /opt/ && cd /opt
 sudo tar -xvf idea-2016.1.1.tar.gz
 ```
 
-:point_left: Previous | :book: Home :book: |  Next :point_right:
-:---|:---:|---:
- [**Installing DevTools and modern web browsers**](Ubuntu-Installing-DevTools) | [**Table of Contents**](Setting-Up-Ubuntu-For-Programming) | [**Alternatives to popular Windows and Mac software**](Ubuntu-Software-Alternatives)
\ No newline at end of file
+:point_left: Previous                                                         |                     :book: Home :book:                     |                                                                   Next :point_right:
+:---------------------------------------------------------------------------- | :--------------------------------------------------------: | -----------------------------------------------------------------------------------:
```
Comment (abhisekp) [inline Ubuntu-Downloading-Text-Editors-and-IDEs.md]: Please revert this back

File: Ubuntu-Downloading-Text-Editors-and-IDEs.md [hunk 4]
```diff
@@ -62,13 +64,13 @@ You can now drag this file to the launcher, enabling you to start Eclipse.
 
 **Installing Intellij IDEA:**
 
-Download the latest version of Intellij IDEA: https://www.jetbrains.com/idea/download/#section=linux
+Download the latest version of Intellij IDEA: [Intellij](https://www.jetbrains.com/idea/download/#section=linux)
```
Comment (abhisekp) [inline Ubuntu-Downloading-Text-Editors-and-IDEs.md]: Download the latest version of [**Intellij IDEA**](https://www.jetbrains.com/idea/download)

```diff
- Download the latest version of Intellij IDEA: [Intellij](https://www.jetbrains.com/idea/download/#section=linux)
+ Download the latest version of [**Intellij IDEA**](https://www.jetbrains.com/idea/download)
```

File: Ubuntu-Installing-DevTools.md [hunk 5]
```diff
@@ -4,42 +4,48 @@ We show you how to install various programming langauges, devtools and web brows
...
 sudo apt-get update
 sudo apt-get install firefox
 ```
 
-:point_left: Previous | :book: Home :book: |  Next :point_right:
-:---|:---:|---:
- [**Jazzing up the Terminal**](Ubuntu-Jazzing-up-the-Terminal) | [**Table of Contents**](Setting-Up-Ubuntu-For-Programming) | [**Downloading Text Editors and IDEs**](Ubuntu-Downloading-Text-Editors-and-IDEs)
\ No newline at end of file
+:point_left: Previous                                         |                     :book: Home :book:                     |                                                                Next :point_right:
+:------------------------------------------------------------ | :--------------------------------------------------------: | --------------------------------------------------------------------------------:
```
Comment (abhisekp) [inline Ubuntu-Installing-DevTools.md]: Please revert this

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

I have done a check of the Semantics and not the content of the articles themselves.
This can be merged.

**Comment by theflametrooper** [comment]
@raisedadead I was assigned to this PR :D

**Pull request merged**